src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.cs

```diff
@@ -257,6 +257,44 @@ async Task Test(bool expectOutput)
             }
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNetCore))]
+        public async Task ListOfTupleTest()
+        {
+            string source = """
+                using Microsoft.Extensions.Configuration;
+                using System;
+                using System.Collections.Generic;
+
+                public class Program
+                {
+                    public static void Main()
+                    {
+                        ConfigurationBuilder configurationBuilder = new();
+                        IConfiguration config = configurationBuilder.Build();
+
+                        var settingsSection = config.GetSection("Settings");
+
+                        Settings options = settingsSection.Get<Settings>()!;
+                    }
+                }
+
+                public class Settings
+                {
+                    public List<(string Item1, string? Item2)>? Items { get; set; }
+                    // public List<string>? Items { get; set; }
```

Comment:
nit: 
```suggestion
```

Reply:
Interesting, looks I already fixed it when you were adding this comment :-)